tests/test_time.py

```diff
@@ -58,6 +58,58 @@ def test_map_years(self):
         )
         assert np.array_equal(years, expected)
 
+    def test_map_to_data(self):
+        # test the edge value when the input could not cover the anchor date
+        cal = AdventCalendar(anchor_date=(10, 15), freq='180d')
+        # single year covered
+        time_index = pd.date_range('20191020', '20211001', freq='60d')
```

Comment:
Tests look good. I think it's good practice to make one test function per test, now you have multiple test cases in one function. I'd recommend splitting them up.

Reply:
Thanks for the comment. I split those test cases now and rename each test descriptively.